tcf_website/views/review.py

```diff
@@ -83,7 +73,32 @@ def new_review(request):
     return render(request, 'reviews/new_review.html')
 
 
+@login_required()
+def check_duplicate(request):
```

Comment:
This is something that doesn't matter for prod, but if I log out and manually to to `http://localhost:8000/reviews/check_duplicate`, it first prompts me to log in (which is good) but then crashes with this error:

![image](https://user-images.githubusercontent.com/55100084/110561927-91c80300-8116-11eb-92ae-fff64ab82738.png)

Realistically this isn't something that we should ever have to worry about, but it's probably good to just add some basic error checking to catch this.